skfda/exploratory/stats/_stats.py

```diff
@@ -126,7 +126,8 @@ def geometric_median(
     r"""Compute the geometric median.
 
     The sample geometric median is the point that minimizes the :math:`L_1`
-    norm of the vector of distances to all observations:
+    norm of the vector of distances to all observations
+    :footcite:`gervini_2008_estimation`:
```

Comment:
Can't you use https://sphinxcontrib-bibtex.readthedocs.io/en/latest/usage.html#listing-citation-keys instead of adding a cite?